src/__tests__/App.test.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { Provider } from 'react-redux';
+import store from '../redux/store';
+import App from '../App';
+
+it('App component test', () => {
+  const tree = renderer
+    .create(
+      <Provider store={store}>
+        <App />
+      </Provider>,
+    )
+    .toJSON();
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
- [ ] Hi, I would advise you to work on the test again. the test did not pass.

![1](https://user-images.githubusercontent.com/63972670/227644196-7bd9c836-dc5b-4a78-992e-06a86617be2f.PNG)